RevenueCatUI/Templates/Components/Package/PackageGroupComponentViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PackageGroupComponentViewModel.swift
+//
+//  Created by Josh Holtz on 9/27/24.
+
+import Foundation
+import RevenueCat
+
+// swiftlint:disable missing_docs
+
+#if PAYWALL_COMPONENTS
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+public class PackageGroupComponentViewModel {
```

Comment:
I think it can probably be public since it's behind the PAYWALL_COMPONENTS compiler flag. If the flag didn't exist, I'd say that it would need to be `internal` until the feature is complete, but devs won't see this unless they enable the `PAYWALL_COMPONENTS` flag

Reply:
Right, but we should try to make it internal from the beginning (unless there is a reason to keep them public) so we don't forget :)